operation-bench/bench.c

```diff
@@ -0,0 +1,68 @@
+#ifndef WIN32
+#include <sys/time.h>
+#endif
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <math.h>
+#include "tsc_x86.h"
+
+#define N 1e7
```

Comment:
Naming could be more descriptive, especially if it's something non trivial as "size of the arrays that store the data on computations".